pina/solvers/pinn.py

```diff
@@ -128,6 +143,13 @@ def training_step(self, batch, batch_idx):
 
             condition_losses.append(loss * condition.data_weight)
 
+        # clamp inferred parameters to their domain
```

Comment:
does `torch.clamp_` remove the `LabelTensor` label? Can we check it and in case override the method in `labeltensor.py`